mpas_analysis/ocean/ohc_timeseries.py

```diff
@@ -72,20 +73,22 @@ def ohc_timeseries(config, streamMap=None, variableMap=None):
 
     # get a list of timeSeriesStats output files from the streams file,
     # reading only those that are between the start and end dates
-    #startDate = config.get('time', 'timeseries_start_date')
-    startDate = '0001-01-01_00:00:00'
+    startDate = config.get('time', 'timeseries_start_date')
     endDate = config.get('time', 'timeseries_end_date')
     streamName = streams.find_stream(streamMap['timeSeriesStats'])
     infiles = streams.readpath(streamName, startDate=startDate,
                                endDate=endDate)
     print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
-    print '  Warning: OHC time series always start from simulation year 1'
 
     # Define/read in general variables
     print '  Read in depth and compute specific depth indexes...'
     f = netcdf_dataset(inputfile, mode='r')
     # reference depth [m]
     depth = f.variables["refBottomDepth"][:]
+    # simulation start time
+    simStartTime = netCDF4.chartostring(f.variables["simulationStartTime"][:])
```

Comment:
For the same purpose, @pwolfram and I had used something like `''.join(f.variables["simulationStartTime"][:]).strip()` but it's much less obvious to someone reading the code what that might do.
https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/shared/mpas_xarray/mpas_xarray.py#L167